terraform/modules/tf_stream_alert_kinesis_streams/main.tf

```diff
@@ -19,3 +19,37 @@ resource "aws_kinesis_stream" "stream_alert_stream" {
     Cluster = "${var.cluster_name}"
   }
 }
+
+// IAM User: stream_alert user for systems to send data to the stream
+resource "aws_iam_user" "stream_alert" {
+  name = "stream_alert_writeonly"
```

Comment:
Can you reason about naming this IAM user name here? Is it renamed? I could not find the original name but the code been deleted is [here](https://github.com/airbnb/streamalert/blob/b3d62705293228f7aae1763d4675b7113dc8f17d/terraform/modules/tf_stream_alert_kinesis/users.tf)

Reply:
It's renamed for clarity